### ISSUE TITLE
aria/aria-roledescription_attribute

### ISSUE THREAD

**Issue by zellwk** [accessibilitysupported/a11ysupport.io#94]
Voiceover Mac isn't fully supported. 

Try testing with a generic element like `<div role="group" aria-roledescription="Slide">` Safari Mac doesn't announce "Slide".

**Comment by mfairchild365** [comment]
Good catch. I wonder if naming the element would help. I'll put together some more test cases for this soon.

**Comment by mfairchild365** [comment]
@zellwk I did some more research on this. No screen readers convey any semantics for role="group" when the element is not named. In other words, `<div role="group">` would not be conveyed at all, while `<div role="group" aria-label="im a name">` would convey a role and boundaries. More details: https://a11ysupport.io/tech/aria/group_role

I wouldn't expect `aria-roledescription` to change this behavior. Instead, it will likely only be used if the screen reader would normally convey the role. Does this make sense to you @zellwk?
